core/src/main/kotlin/org/evomaster/core/languagemodel/data/Prompt.kt

```diff
@@ -0,0 +1,14 @@
+package org.evomaster.core.languagemodel.data
+
+class Prompt (
+    val id: String,
+
+    val prompt: String
+) {
+
+    var answer: String? = null
```

Comment:
adding `answer` to `Prompt` might be confusing. better to create a new class `AnsweredPrompt` with fields `prompt: Prompt` and `answer: String`